docs/UserGuide.md

```diff
@@ -559,14 +559,31 @@ There are three `TYPE`s:
 
 `com` refers to Company tab, `app` refers to Application tab, `me` refers to Profile tab.
 
-Example: 
-* `switch me`
+Example:
+
+1. Suppose you want to switch to the application tab. Executing `switch app` will switch the tabs to application tab.
+
+<p align="center"><img src="images/ug-general/switchApp.png" width="70%" height="70%"/></p>
+
+2. Note that the tabs has been changed as well as the cards and display.
```

Comment:
Perhaps you can update it to *have been changed

Reply:
fixed